EventFilter/CSCRawToDigi/plugins/CSCDCCUnpacker.cc

```diff
@@ -9,21 +13,23 @@
 #include "FWCore/Utilities/interface/InputTag.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
```

Comment:
Not really relevant, but it wouldn't be bad grouping together similar includes (i.e. ParameterSet close to ParameterSetDescription)

Reply:
Moved.